### PR TITLE
Update to bidding logic

### PR DESCRIPTION
**Pull request by RobbyG1989**
Adding k/K to numbers
Adding ability to bid with single/multiple digits if current bid is over 1000 (1 = 1000, 998 = 998000, 1000 = 1000, 5000 = 5000)

Cleaned up the commits and changed the logic a bit to avoid confusion on bidding.

### CODE REVIEW

**Comment by RobbyG1989** [comment]
Changed to remove the multiply by 1000 for single/multiple digit bids under 999

**Comment by tg123** [comment]
hmm is there a way to integrate chatgpt with us?
I was also replying to homies that wish this feature: NLU is not strong enough to do this for us

**Comment by RobbyG1989** [comment]
I mean potentially, you'd have to train the model on lua code more frequently to get better results. 

As an aside, i did some testing for a single bid and found that this works:

`if ask >= bid then
					ctx.currentwinner = playerName
					ctx.currentprice = ask * 10000
					ctx.countdown = bf.countdown:GetValue()
                    
                    -- L["Bid price"]
                    SendRaidMessage(L["Bid accept"] .. " " .. item .. " " .. L["Current price"] .. " >>" .. GetMoneyStringL(ctx.currentprice) .. "<< ".. (ctx.pause and "" or L["Time left"] .. " " .. (SECOND_ONELETTER_ABBR:format(ctx.countdown))))
                elseif bid > 999 and ask < 999 then
					ask = ask * 1000
					ctx.currentwinner = playerName
					ctx.currentprice = ask * 10000
					ctx.countdown = bf.countdown:GetValue()
                    
                    -- L["Bid price"]
                    SendRaidMessage(L["Bid accept"] .. " " .. item .. " " .. L["Current price"] .. " >>" .. GetMoneyStringL(ctx.currentprice) .. "<< ".. (ctx.pause and "" or L["Time left"] .. " " .. (SECOND_ONELETTER_ABBR:format(ctx.countdown))))
				else
                    SendRaidMessage(L["Bid denied"] .. " " .. item .. " " .. L["Must bid higher than"] .. " " .. GetMoneyStringL(bid * 10000))
                end`

It allows for single bids like 1,2,5,10,15,ect if the starting bid and the current bid is under 1000.

File: gui.lua [hunk 1]
```diff
@@ -672,28 +672,47 @@ function GUI:Init()
...
+			local item = currentitem()
+
+			if ask >= bid then
+				ctx.currentwinner = playerName
+				ctx.currentprice = ask * 10000
+				ctx.countdown = bf.countdown:GetValue()
+
+				-- L["Bid price"]
+				SendRaidMessage(L["Bid accept"] .. " " .. item .. " " .. L["Current price"] .. " >>" .. GetMoneyStringL(ctx.currentprice) .. "<< ".. (ctx.pause and "" or L["Time left"] .. " " .. (SECOND_ONELETTER_ABBR:format(ctx.countdown))))
+			elseif bid > 999 and ask < 999 then
```
Comment (tg123) [inline gui.lua:705]: what if make this an option and user could set `999` via ui?
no hardcoded value

also should apply to `ask *1000`

**Pull request closed without merging**